blocks/library/html/index.js

```diff
@@ -33,6 +33,10 @@ registerBlockType( 'core/html', {
 
 	supportHTML: false,
 
+	supports: {
+		customClassName: false,
```

Comment:
Can we now drop the `className: false` from this and other revised blocks?

Reply:
We can't, it controls the generated className which we should address separately.